.github/workflows/integration.yml

```diff
@@ -143,6 +61,11 @@ jobs:
         if: steps.cache-venv.outputs.cache-hit != 'true'
         working-directory: ./api
         run: |
+          # Install JDK
```

Comment:
Was the JDK included in the python base image we used previously but not in our base image?

Reply:
Yes! I think we should move it to the base image. I can write a task for that.

Reply:
https://github.com/bcgov/wps/issues/4509